api/routes/user.py

```diff
@@ -396,3 +396,36 @@ def update_favorite_jobs(
 
     db.commit()
     log(log.INFO, "User [%s] successfully updated favorite job list", current_user.id)
+
+
+@user_router.put(
+    "/favorite-expert/{expert_uuid}",
+    status_code=status.HTTP_204_NO_CONTENT,
+    responses={
+        status.HTTP_404_NOT_FOUND: {"description": "Job not found"},
+    },
+)
+def update_favorite_experts(
+    expert_uuid: str,
+    db: Session = Depends(get_db),
+    current_user: m.User = Depends(get_current_user),
+):
+    """Add or remove experts from favorite list"""
+
+    expert = db.scalar(sa.select(m.User).where(m.User.uuid == expert_uuid))
+
+    if not expert:
+        log(log.ERROR, "User [%s] not found", expert_uuid)
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="User not found")
+
+    if expert.id == current_user.id:
+        log(log.ERROR, "User [%s] can't add himself to favorite list", current_user.id)
+        raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail="User can't add himself to favorite list")
+
+    if expert not in current_user.favorite_experts:
+        current_user.favorite_experts.append(expert)
+    else:
+        current_user.favorite_experts.remove(expert)

```

Comment:
I'd consider having separate PUT and DELETE endpoints for there actions. Calling this endpoint several times may cause unpredictable results